spec/factories/event.rb

```diff
@@ -13,6 +13,7 @@
     status_id { 1 }
     status_publish_event { 1 }
     status_publish_event_dates { 1 }
+    form_master_num { 'oh_jfs_04221' }
```

Comment:
Should this be generated so that not every test relies on a static form? You could make it similar to https://github.com/midohiofoodbank/freshtrak-pantry-finder-api/blob/62b808162169468fd2d770b11062111e232ae351/spec/factories/event.rb#L35

Reply:
If you mean adding form to event factory, I will try that.  It wasn't working with my previous less than optimal associations so I gave up on it.

Reply:
Adding form to the event factory seemed to cause some kind of circular reference and I get:

Form belongs to an event
     Failure/Error: zip { agency.zip }
    
     ActiveRecord::RecordNotUnique:
       Mysql2::Error: Duplicate entry '7202' for key 'foodbanks_mini.PRIMARY'

which seems like some kind of circular condition. 

Then when I remove the event from the form factory, I get:

Form belongs to an event
     Failure/Error: expect(form.event).to be_an_instance_of(Event)
       expected nil to be an instance of Event

Reply:
You're right this association is kind of annoying. What if you just leave the `form_master_num` off of this factory and create it manually for the few tests where it's required.
For example:
```
form = create(:form)
event = create(:event, form_master_num: form.form_master_num)
```

Reply:
Missed this comment before I did last commit.  Will be added to next commit